cidc_api/models.py

```diff
@@ -561,6 +561,15 @@ def create(
 
         return job
 
+    @staticmethod
+    @with_default_session
+    def merge_extra_metadata(job_id, artifact_uuid, file, session):
+
+        job = AssayUploads.find_by_id(job_id, session = session)
+        prism.merge_artifact_extra_metadata(job.assay_patch, artifact_uuid, job.assay_type, file)
```

Comment:
so how does prism call result gets saved? am I missing something or is it ignored here? So we need to save it?

Also I would make it a bulk method accepting multiple uuids/files in one, so we won't have to commit multiple times (which seems wrong)